example/driver/simpledriver.go

```diff
@@ -74,11 +78,55 @@ func (s *SimpleDriver) Initialize(lc logger.LoggingClient, asyncCh chan<- *dsMod
 	s.lc = lc
 	s.asyncCh = asyncCh
 	s.deviceCh = deviceCh
+	s.serviceConfig = &config.ServiceConfig{}
+
+	ds := service.RunningService()
+
+	if err := ds.LoadCustomConfig(s.serviceConfig, "SimpleCustom"); err != nil {
+		return fmt.Errorf("unable to load 'SimpleCustom' custom configuration: %s", err.Error())
+	}
+
+	lc.Infof("Custom config is: %v", s.serviceConfig.SimpleCustom)
+
+	if err := s.serviceConfig.SimpleCustom.Validate(); err != nil {
+		return fmt.Errorf("'SimpleCustom' custom configuration validation failed: %s", err.Error())
+	}
+
+	if err := ds.ListenForCustomConfigChanges(
+		&s.serviceConfig.SimpleCustom.Writable,
+		"SimpleCustom/Writable", s.ProcessCustomConfigChanges); err != nil {
+		return fmt.Errorf("unable to listen for changes for 'SimpleCustom.Writable' custom configuration: %s", err.Error())
+	}
+
 	return nil
 }
 
+// ProcessCustomConfigChanges ...
+func (s *SimpleDriver) ProcessCustomConfigChanges(rawWritableConfig interface{}) {
+	updated, ok := rawWritableConfig.(*config.SimpleWritable)
+	if !ok {
+		s.lc.Error("unable to process custom config updates: Can not cast raw config to type 'SimpleWritable'")
+		return
+	}
+
+	s.lc.Info("Received configuration updates for 'SimpleCustom.Writable' section")
+
+	previous := s.serviceConfig.SimpleCustom.Writable
+	s.serviceConfig.SimpleCustom.Writable = *updated
+
+	if reflect.DeepEqual(previous, *updated) {
+		s.lc.Info("No changes detected")
+		return
+	}
+
+	if previous.DiscoverSleepDurationSecs != updated.DiscoverSleepDurationSecs {
```

Comment:
Given that you just did a `DeepEqual` check, I don't think it's possible for these two values to be equal here, so why check? I think your intention was to alert developers that if their `Writable` stanza included more than one field, then using `DeepEqual` makes sense, and there may be cases where some fields remain unchanged? Maybe a comment explaining this would be helpful here?

Reply:
It is just an example of how to check, so if you have multiple setting in the `Writable` section. Comment add